new-feature.txt

```diff
@@ -0,0 +1 @@
+This file was craeted in feature1 brach
```

Comment:
it's really good that you have decided to add theses lines of code

Reply:
Thanks